Modules/_datetimemodule.c

```diff
@@ -1664,6 +1664,12 @@ multiply_float_timedelta(PyObject *floatobj, PyDateTime_Delta *delta)
     ratio = _PyObject_CallMethodId(floatobj, &PyId_as_integer_ratio, NULL);
     if (ratio == NULL)
         goto error;
+    if (!PyTuple_Check(ratio)) {
+        PyErr_SetString(PyExc_TypeError,
+                        "Can't multiply timedelta object by float with "
```

Comment:
It looks like other error messages in this file start with a lowercase letter.  Let's keep the style consistent.

I also wonder if a better message would be "unexpected return type from as_integer_ratio(): expected tuple, got %s".